pkg/cmd/provisioning/aws/create_identity_provider.go

```diff
@@ -11,14 +11,15 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
-	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
 	"io/ioutil"
 	"log"
 	"net/url"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+
```

Comment:
this import should come last, along with other openshift import

Reply:
ahh...VS Code...i almost with it wouldn't auto-import things